pontoon/base/views.py

```diff
@@ -685,6 +685,7 @@ def get_users(request):
                 "gravatar": u.gravatar_url(44),
                 "name": u.name_or_email,
                 "url": u.profile_url,
+                "username": u.profile.username,
```

Comment:
This will result in N additional DB queries, where N is the number of `users` (currently close to 20k on production), which makes the `/get-users/` request quite slow compared to its current performance.

This happens because the username is stored in a separate table (`UserProfile` rather than `User`). The issue can be mitigated by using [`prefetch_related()`](https://docs.djangoproject.com/en/5.0/ref/models/querysets/#prefetch-related).

You can debug DB queries using [`debug_sql()`](https://github.com/mozilla/pontoon/blob/main/pontoon/dev/utils.py#L5).

Reply:
That's a million times faster! Thank you for pointing that out